.eslintignore

```diff
@@ -0,0 +1 @@
+**/babel-plugin-glaze/__test__/**
```

Comment:
I had a lot of eslint errors when trying to commit. Turned out that lots of them were coming from the test files. Don't know If it's the best way to ignore them but at least it worked :P

Reply:
ESLint is [configured](https://github.com/kripod/glaze/blob/08cc3fd809df7db2af68767f2c618a109b2a523c/package.json#L12) to only ignore the files specified in `.gitignore` Unfortunately, the `.eslintignore` file will have no effect.